Blogger.API/Infrastructure/Respositories/BlogRepository.cs

```diff
@@ -25,12 +25,14 @@ public async Task CreateAsync(Blog blog)
 
         public async Task<List<Blog>> GetAllAsync()
         {
-            return await _dbContext.Blogs.Where(b => b.IsDeleted == false).ToListAsync();
+            return await _dbContext.Blogs
+                            .Where(b => b.IsDeleted == false)
```

Comment:
b.IsDeleted == false is equivalent to: !b.IsDeleted

Basically add the exclamation mark at the start. Use the 2nd approach cause it is cleaner and less verbose

Reply:
Also why so much indentation? Just press tab once